kubos/utils/git_utils.py

```diff
@@ -32,23 +32,46 @@ def get_tag_list(repo):
     tags = repo.tags
     tag_list = []
     for tag in tags:
-        tag_list.append(tag)
+        tag_list.insert(0, tag.name)
     return tag_list
 
 
-def print_tag_list(tag_list):
+def filter_cd_generated_tags(display_num, tag_list):
+    '''
+    With every merge to Master of the Kubos repo theres the CD config generates
+    and releases a new tag/release..
+    This filters only the most recent display_num number of release tags.
+    '''
+    filtered_tags = []
+    major_version = re.compile('v?\d+\.\d+\.\d+')
+    minor_version = re.compile('v?\d+\.\d+\.\d+\.\d+')
```

Comment:
Continuing with the renaming. Could these instead be something like 'release_version' and 'cd_build_version'?